<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1024</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QPushButton" name="pushButton_return">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>470</y>
     <width>161</width>
     <height>91</height>
    </rect>
   </property>
   <property name="text">
    <string>Retour</string>
   </property>
  </widget>
  <widget class="QSlider" name="slider">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>330</y>
     <width>911</width>
     <height>111</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <property name="invertedAppearance">
    <bool>false</bool>
   </property>
   <property name="invertedControls">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QLabel" name="label_Title">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>30</y>
     <width>611</width>
     <height>71</height>
    </rect>
   </property>
   <property name="text">
    <string>Maintenance</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_home">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>120</y>
     <width>281</width>
     <height>101</height>
    </rect>
   </property>
   <property name="text">
    <string>Home</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_bottle">
   <property name="geometry">
    <rect>
     <x>590</x>
     <y>110</y>
     <width>281</width>
     <height>101</height>
    </rect>
   </property>
   <property name="text">
    <string>Bouteilles</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_bottom_screen">
   <property name="geometry">
    <rect>
     <x>380</x>
     <y>480</y>
     <width>501</width>
     <height>91</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_send">
   <property name="geometry">
    <rect>
     <x>750</x>
     <y>470</y>
     <width>201</width>
     <height>91</height>
    </rect>
   </property>
   <property name="text">
    <string>Envoyer</string>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBox_axis">
   <property name="geometry">
    <rect>
     <x>420</x>
     <y>490</y>
     <width>131</width>
     <height>71</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_axis">
   <property name="geometry">
    <rect>
     <x>440</x>
     <y>310</y>
     <width>101</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>axis Value</string>
   </property>
  </widget>
  <zorder>label_bottom_screen</zorder>
  <zorder>pushButton_return</zorder>
  <zorder>slider</zorder>
  <zorder>label_Title</zorder>
  <zorder>pushButton_home</zorder>
  <zorder>pushButton_bottle</zorder>
  <zorder>pushButton_send</zorder>
  <zorder>comboBox_axis</zorder>
  <zorder>label_axis</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
